themes-default/slim/views/config_subtitles.mako

```diff
@@ -14,20 +15,76 @@ const startVue = () => {
         },
         data() {
             return {
-                header: 'Subtitles'
+                header: 'Subtitles',
+                services: JSON.parse('${json.dumps(subtitles.sorted_service_list())}')
             };
         },
+        computed: {
+            enabledServices() {
+                const { services } = this;
+                return services.map(service => service.name + ':' + (service.enabled ? '1' : '0')).join(' ');
+            }
+        },
         mounted() {
             $("#subtitles_languages").tokenInput([${','.join("{\"id\": \"" + code + "\", name: \"" + subtitles.name_from_code(code) + "\"}" for code in subtitles.subtitle_code_filter())}], {
-                method: "POST",
-                hintText: "Write to search a language and select it",
+                method: 'POST',
+                hintText: 'Write to search a language and select it',
                 preventDuplicates: true,
-                prePopulate: [${','.join("{\"id\": \"" + code + "\", name: \"" + subtitles.name_from_code(code) + "\"}" for code in subtitles.wanted_languages())}],
-                resultsFormatter: (item) => "<li><img src='images/subtitles/flags/" + item.id + ".png' onError='this.onerror=null;this.src=\"images/flags/unknown.png\";' style='vertical-align: middle !important;' /> " + item.name + "</li>",
-                tokenFormatter: (item) => "<li><img src='images/subtitles/flags/" + item.id + ".png' onError='this.onerror=null;this.src=\"images/flags/unknown.png\";' style='vertical-align: middle !important;' /> " + item.name + "</li>"
+                prePopulate: [${','.join('{"id": "' + code + '", name: "' + subtitles.name_from_code(code) + '"}' for code in subtitles.wanted_languages())}],
+                resultsFormatter: item => "<li><img src='images/subtitles/flags/" + item.id + ".png' onError='this.onerror=null;this.src=\"images/flags/unknown.png\";' style='vertical-align: middle !important;' /> " + item.name + "</li>",
+                tokenFormatter: item => "<li><img src='images/subtitles/flags/" + item.id + ".png' onError='this.onerror=null;this.src=\"images/flags/unknown.png\";' style='vertical-align: middle !important;' /> " + item.name + "</li>"
             });
             $('#config-components').tabs();
             $('#subtitles_dir').fileBrowser({ title: 'Select Subtitles Download Directory' });
+            $('#service_order_list').disableSelection();
+
+            //
+            // Below this line is stuff that may or may not be doing anything?
```

Comment:
I assume this used to be the way subtitle services were rendered, much like on the search providers page.

The subtitle services list used to be sortable. Now I see you removed much of that code (due to console errors?)
Does the subtitle service matter in the context of searching for subtitles? If so, need to re-add that.
If not, need to remove that little red icon that indicates sorting, and change the caption:
**"Check off and drag the plugins into the order you want them to be used."**